src/modules/Dashboard/components/CardImage/CardImage.jsx

```diff
@@ -84,7 +99,11 @@ const CardImage = ({ deleteCurrentImageAction, image }) => {
                 <Card.Footer>
                     <Row>
                         <Col xs={6}>
-                            <Button variant="primary">
+                            <Button
```

Comment:
Add the `download` attribute to `a` tag. See [here](https://www.w3schools.com/howto/howto_html_download_link.asp)